src/scenes/ProgressReports/EditForm/ProgressReportDrawerHeader.tsx

```diff
@@ -51,54 +39,71 @@ export const ProgressReportDrawerHeader = ({ report }: ReportProp) => {
       >
         <Card elevation={0} variant="outlined" sx={{ px: 2, py: 1 }}>
           <Typography variant="subtitle2" color="text.gray">
-            Location
+            Primary Location
           </Typography>
-          {project.primaryLocation.value?.name.canRead && (
-            <div css={{ display: 'flex' }}>
-              <Typography variant="body1">
-                {project.primaryLocation.value.name.value}
-              </Typography>
-              {project.fieldRegion.value && (
-                <span
-                  css={(theme) => ({
-                    marginRight: theme.spacing(1),
-                    marginLeft: theme.spacing(1),
-                  })}
-                >
-                  |
-                </span>
-              )}
-              <Typography variant="body1">
-                {project.fieldRegion.value?.name.value}
-              </Typography>
-            </div>
-          )}
+          <Box sx={{ display: 'flex', alignItems: 'center', ml: -0.3 }}>
+            <MapPinIcon color="info" />
+            {project.primaryLocation.value?.name.canRead && (
+              <div css={{ display: 'flex' }}>
+                <Typography variant="body1" sx={{ ml: 0.8 }}>
+                  {project.primaryLocation.value.name.value}
+                </Typography>
+                {project.fieldRegion.value && (
+                  <span
+                    css={(theme) => ({
+                      marginRight: theme.spacing(1),
+                      marginLeft: theme.spacing(1),
+                    })}
+                  >
+                    |
+                  </span>
+                )}
+                <Typography variant="body1">
+                  {project.fieldRegion.value?.name.value}
+                </Typography>
+              </div>
+            )}
+          </Box>
         </Card>
         <Card elevation={0} variant="outlined" sx={{ px: 2, py: 1 }}>
           <Typography variant="subtitle2" color="text.gray">
             Sensitivity
           </Typography>
-          <Typography variant="body1" sx={{ textTransform: 'uppercase' }}>
-            {sensitivity}
+          <Box sx={{ display: 'flex', alignItems: 'center', ml: -0.6 }}>
             <SensitivityIcon
               value={sensitivity}
               sx={{
                 height: 16,
               }}
             />
-          </Typography>
+            <Typography
+              variant="body1"
+              sx={{ textTransform: 'uppercase', mt: 0.2 }}
+            >
+              {sensitivity}
+            </Typography>
+          </Box>
         </Card>
       </Box>
+      <Box sx={{ display: 'flex', mt: 2 }}>
+        <Typography variant="h5">{project.name.value}</Typography>
+        <Divider
+          orientation="vertical"
+          flexItem
+          sx={{ mx: 2, borderRightColor: 'black', borderRightWidth: 2 }}
+        />
```

Comment:
If we want the divider to look like like the text I would just use the pipe character instead.
```tsx
<Typography variant="h5">
{project.name.value}
|
{language.value?.displayName.value}
</Typography>
```
(still might need to tweak spacing, with `{" "}` or maybe `&nbsp;`)

Otherwise I'd use the `Divider` component as is. If we don't like the way the Divider looks we can change it in the theme.
Using hardcoded colors is a code smell. The text next to it is not `"black"` either, it's a shade of black.

I'd also use a `Stack` here instead. Check out their [example which includes dividers](https://mui.com/material-ui/react-stack/#dividers).

Reply:
I updated this to use the Stack and divider like the link you provided shows.   I think it made the spacing look better too.